plenum/common/messages/fields.py

```diff
@@ -705,13 +705,12 @@ def _specific_validation(self, val):
 
 
 class BatchIDField(FieldBase):
-    _base_types = (list, tuple)
+    _base_types = (list, tuple, dict)
 
     def _specific_validation(self, val):
         if len(val) != 4:
             return 'should have size of 4'
-
-        bid = BatchID(*val)
+        bid = BatchID(**val) if isinstance(val, dict) else BatchID(*val)
```

Comment:
1. I don't think we need to support both dict and tuple when serialized
2. This can throw, for example if we receive dict with some keys not in BatchID field list

Reply:
done